tree/ntuple/v7/src/RDaos.cxx

```diff
@@ -64,32 +64,40 @@ std::string ROOT::Experimental::Detail::RDaosObject::ObjClassId::ToString() cons
 }
 
 ROOT::Experimental::Detail::RDaosObject::FetchUpdateArgs::FetchUpdateArgs(FetchUpdateArgs &&fua)
-   : fDkey(fua.fDkey), fAkey(fua.fAkey), fIods{fua.fIods[0]}, fSgls{fua.fSgls[0]}, fIovs(std::move(fua.fIovs)),
-     fEvent(std::move(fua.fEvent))
+   : fDkey(fua.fDkey), fAkeys(std::move(fua.fAkeys)), fIods(std::move(fua.fIods)), fSgls(std::move(fua.fSgls)),
+     fIovs(std::move(fua.fIovs)), fEvent(std::move(fua.fEvent))
```

Comment:
Note that if we use `std::span<...>` for the list of attribute keys and iovs, `std::move()` will have to be removed for those (also below).